src/commands/dsyms/__tests__/upload.test.ts

```diff
@@ -1,9 +1,8 @@
-// tslint:disable: no-string-literal
-import {Cli} from 'clipanion/lib/advanced'
 import {existsSync, promises} from 'fs'
-import glob from 'glob'
 import {EOL, platform} from 'os'
 import path from 'path'
+import glob from 'glob'
+import {Cli} from 'clipanion/lib/advanced'
```

Comment:
Could the rules be changed to have a blank line between import groups?
Else it is particularly strange to have linter force us to have `glob` right after `path`

Reply:
You can try this rule configuration to mirror what we have in other repos:
```js
'import/order': [
  'error',
  {
    alphabetize: {order: 'asc', caseInsensitive: true},
    'newlines-between': 'always',
    groups: [['builtin', 'object'], 'type', 'external', 'internal', 'parent', ['sibling', 'index']],
    // set different groups for 5 different import levels ('../../../../**', '../../../**', ...)
    pathGroups: [
      ...Array.from({length: 5}).map((_, index) => ({
        pattern: `${'../'.repeat(5 + 1 - index)}**`,
        group: 'parent',
        position: 'before',
      })),
    ],
  },
],
```